internal/util/fileop.go

```diff
@@ -102,6 +102,11 @@ func (this *FileOperation) Validate() error {
 			return fmt.Errorf("Argument at index %d ('%s') cannot point just to the current directory. File operation paths must point to a dirent inside the current directory tree.", i, parts[i])
 		}
 
+		_, err := filepath.Match(path, "")
```

Comment:
A non-glob (e.g., dest) can have a path that is valid but looks like an invalid glob pattern.